### DISCUSSION TITLE
Creating custom plugin that's supports subscript

### DISCUSSION THREAD

**Discussion by Lambdaderuta** [remarkjs/remark#818] [Q&A] [answered]
Hi! having troubles with subscript. I using React markdown lib. Unfortunately the remark plugin that was supposed to solve my problem is broken [plugin](https://github.com/zestedesavoir/zmarkdown/tree/HEAD/packages/remark-sub-super). I was trying create custom plugin to support subscript but i was stuck with cases when inside subcript has another markdown  strong, emhasis etc.

Can somebody point me on links or examples how to achive desired behavior?

**Comment by wooorm** [comment, marked as answer]
The [§ Extending markdown](https://github.com/micromark/micromark#extending-markdown) and [§ Creating a micromark extension](https://github.com/micromark/micromark#creating-a-micromark-extension) sections in the micromark readme will give you the background you need.
Generally, I’d say, don’t do it.

**Reply by Lambdaderuta** [reply]
Thanks for quick response! I will probably have to give up this idea. Can't mark your comment as answer cause of 422 http error :(

**Reply by ChristianMurphy** [reply]
to add on.

> Unfortunately the remark plugin that was supposed to solve my problem is broken plugin.

This might be related to: https://github.com/zestedesavoir/zmarkdown/issues/416 ?
If not it may be worth raising an issue at zmarkdown so the maintainer(s) are aware.

**Reply by Lambdaderuta** [reply]
Thanks! Just opened new issue https://github.com/zestedesavoir/zmarkdown/issues/445
